jobs/execute.ts

```diff
@@ -0,0 +1,105 @@
+import { JobName, allJobs } from './list';
+import tracer from '../common/logger/tracing';
+import { getLogger } from '../common/logger/logger';
+import { metrics, metricsRouter } from '../common/logger/metrics';
+import express from 'express';
+import http from 'http';
+import { prisma } from '../common/prisma';
+import { Prisma } from '@prisma/client';
+
+const logger = getLogger('Job Execution');
+
+async function startMetricsServer() {
+    const app = express();
+    app.use('/metrics', metricsRouter);
+
+    const port = process.env.PORT || 5100;
+
+    const server = http.createServer(app);
+
+    // Start listening
+    await new Promise<void>((res) => server.listen(port, res));
+    logger.info(`Server listening on port ${port}`);
+}
+
+if (process.env.METRICS_SERVER_ENABLED === 'true') {
+    startMetricsServer().catch((e) => logger.error('Failed to setup metrics server', e));
+}
```

Comment:
This part should stay in the scheduler. When running jobs from the web, it will use the metrics endpoint [from there](https://github.com/corona-school/backend/blob/master/web/server.ts#L105).

Reply:
Ah I see. Moved it to jobs/index.ts which is supposed to start that Dyno